.github/workflows/spellcheck.yml

```diff
@@ -0,0 +1,26 @@
+name: Run spellchecker for latex and markdown files
+on: 
+  push:
+    paths:
+      - '**.md'
+      - '**.tex'
+      - '.github/workflows/spellcheck.yml'
+      - 'scripts/spellcheck.sh'
+
+jobs:
+  spellcheck:
+    runs-on: ubuntu-latest
+    name: Spellchecker
+
+    steps:
+    - name: Download repo
+      uses: actions/checkout@v2
+
+    - name: Install spellchecker tool
+      run: sudo apt install aspell aspell-es aspell-en
+
+    - name: Add execution permissions to spellchecker script
+      run: chmod +x scripts/spellcheck.sh
```

Comment:
Just do it on the repo.

Reply:
👍

Reply:
I think I will leave it as it is, because for some reason it will bother me redundantly giving execution permissions every time I want to spellcheck something on local.

If it ever causes any problems I will move it to the makefile as a workaround, but I don't think it hurts having it handled like this.